bugwarrior/services/gitea.py

```diff
@@ -0,0 +1,564 @@
+#/home/joybuke/Documents/ComputerScience/Projects/Personal/bugwarrior/bugwarrior/services coding: utf-8
+# gitea.py
+"""Bugwarrior service support class for Gitea
+
+Available classes:
+- GiteaClient(ServiceClient): Constructs Gitea API strings
+- GiteaIssue(Issue): TaskWarrior Interface
+- GiteaService(IssueService): Engine for firing off requests
+
+Todo:
+    * Add token support
+    * Flesh out more features offered by gitea api
+"""
+from builtins import filter
+import logging
+import pathlib
+import re
+import six
+import sys
+from urllib.parse import urlparse
+from urllib.parse import quote_plus
+
+import requests
+from six.moves.urllib.parse import quote_plus
+import typing_extensions
+from jinja2 import Template
+
+from bugwarrior import config
+from bugwarrior.services import IssueService, Issue, ServiceClient
+
+log = logging.getLogger(__name__)  # pylint: disable-msg=C0103
+
+class GiteaConfig(config.ServiceConfig):
+    service: typing_extensions.Literal['gitea']
+
+    host = "gitea.com"
+    login: str
+    token: str
+    login: str
+    username: str
+    password: str
+    exclude_repos = []
+    include_repos = []
```

Comment:
```suggestion
    exclude_repos = config.ConfigList([])
    include_repos = config.ConfigList([])
```

Needed for backwards compatibility with INI format.